src/Microsoft.PowerShell.Commands.Utility/commands/utility/Group-Object.cs

```diff
@@ -153,15 +153,15 @@ private static string BuildName(List<ObjectCommandPropertyValue> propValues)
 
                         foreach (object item in propertyValueItems)
                         {
-                            sb.Append(CultureInfo.InvariantCulture, $"{item}, ");
+                            sb.AppendFormat(CultureInfo.InvariantCulture, "{0}, ", item.ToString());
```

Comment:
Root of the issue is a tricky code. Why do we use CultureInfo.InvariantCulture?
```suggestion
                            sb.AppendFormat(CultureInfo.CurrentCulture, $"{item}, ");
```

Reply:
I don't know the history of this code, but using currentculture with interpolated seems fine